CONTRIBUTING.md

```diff
@@ -1,70 +1,2 @@
 # Instructions for Internal Contributors
 > At the moment, we are not accepting external contributions. Please watch this space to know when we open.
-
-### Building locally
-Clone a copy of sfdx-scanner code through git:
-```
-git clone https://github.com/forcedotcom/sfdx-scanner.git
-```
-
-Build first time using yarn:
-```
-cd sfdx-scanner
-yarn --ignore-scripts && yarn build
-```
-
-### Running
-Run any sf scanner command by replacing `sf` with `bin/dev.js` or `bin/dev.cmd` from sfdx-scanner directory. For example, you can invoke `list` command with:
-```
-bin/dev.js scanner rule list
-```
-
-### Making changes
-Checkout a branch copy of `dev` branch. Make your changes and write unit tests in `./test/` directory.
-To run tests, use:
-```
-yarn test
-```
-
-Before pushing your changes and creating a PR, run this to make sure the build would remain stable:
-
-```
-yarn --ignore-scripts && yarn build && yarn test && yarn lint
-```
-
-### Debugging your plugin
-We recommend using the Visual Studio Code (VS Code) IDE for your plugin development. Included in the `.vscode` directory of this plugin is a `launch.json` config file, which allows you to attach a debugger to the node process when running your commands.
-
-To debug the `scanner rule list` command: 
```

Comment:
This file was filled with v4 information. So let's remove the text for now and we'll add it back in before we go GA.